python/ccf/ledger.py

```diff
@@ -944,9 +951,13 @@ def last_verified_txid(self) -> TxID:
 
         :return: :py:class:`ccf.tx_id.TxID`
         """
-        return TxID(
-            self._ledger_validator.last_verified_view,
-            self._ledger_validator.last_verified_seqno,
+        return (
+            TxID(
+                self._ledger_validator.last_verified_view,
+                self._ledger_validator.last_verified_seqno,
+            )
+            if self._ledger_validator
+            else TxID(0, 0)
```

Comment:
I think it'd be better to return `None` here.

Reply:
Or raise, I don't know which is more Pythonic.

Reply:
I went for `None`.